L/libjulia/libjulia@1.7/build_tarballs.jl

```diff
@@ -1,2 +1,3 @@
 include("../common.jl")
-build_julia(ARGS, v"1.7.0-beta3")
+build_julia(ARGS, v"1.6.0"; jllversion=v"1.7.0")
+build_julia(ARGS, v"1.7.0-beta3"; jllversion=v"1.7.0")
```

Comment:
So is Yggdrasil now using Julia 1.7?

What's the plan going forward for JLL versions: when 1.8 is out, we'll make a JLL version 1.8 which contains binaries for 1.6, 1.7, 1.8?

Reply:
Oh and I see this still uses `L/libjulia/libjulia@1.7/build_tarballs.jl` (note the 1.7 in this filename). Do we expect to ever user the builders for older Julia versions again? If not, how about simplifying things further, and getting rid of the `L/libjulia/libjulia@1.*` dirs again. Then this file here would become `L/libjulia/build_tarballs.jl` (and I guess it could be merged with `common.jl`, although there is no strong reason to do so right now)

Reply:
I like the idea of just having a build_tarballs, yes. There is however #2898